src/main/java/de/xm/jdbcexcel/cellwriters/DateCellWriter.java

```diff
@@ -0,0 +1,34 @@
+package de.xm.jdbcexcel.cellwriters;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.Workbook;
+
+import java.util.Date;
+
+public class DateCellWriter extends AbstractCellWriter<Date> {
+
+    private final static String DATE_FORMAT = "dd.MM.yyyy";
+
+    private CellStyle dateCellStyle;
+
+    @Override
+    public int doWriteCell(Workbook workbook, Cell cell, Date cellValue) {
+        cell.setCellValue(cellValue);
+
+        if (dateCellStyle == null) {
+            dateCellStyle = createNewDateCellStyle(workbook);
```

Comment:
I guess a null check is not enogh here, the dateCellStyle depends on the workbook, so the lazy init is only valid if the workbook is the same as the one used to create the cell style

Reply:
The cell writers are scoped to a workbook tab. They are recreated for each tab so this should be fine, I have added a test to confirm this.